app/src/main/ui/src/components/helpers/i18n.tsx

```diff
@@ -0,0 +1,33 @@
+import i18n from "i18next";
+import {Trans} from "react-i18next";
+
+export function translate(_key: string): string;
+export function translate(_key: string, _options: any): string;
+
+export function translate(key: string, options?: any) {
+    if (options) {
+        return i18n.t(key, options) as unknown as string;
+    }
+    return i18n.t(key) as string;
+}
+
+export function getLocale(): string {
+    return i18n.language;
+}
+
+/**
+ * translateWithEnglishWords can be used to mark English words within a non English translation using the tag <en/>
+ * Example: de-Translation has the following key
+ *  key: "Der Button wird zum <en>Logout</en> verwendet"
+ *  output: "Der Button wird zum <span lang="en">Logout</span> verwendet"
+ * @param key within i18n translation
+ * @returns Translation with embedded html tag
+ */
+export function translateWithEnglishWords(key: string, placesholders?: any) {
+    return (
+        <Trans
+            i18nKey={key}
+            values={placesholders}
+            components={{en: <span lang="en"/>}}/>
+    );
+}
```

Comment:
All functions are already included in the new ces-theme:
https://ecosystem.cloudogu.com/scm/repo/ces/ces-theme-tailwind/code/sources/develop/src/functions/i18n.tsx/

Reply:
Alright, I will remove this file then. let me check

Reply:
it seems the included function of the theme does not work with span title, do you have any idea on how to fix it?